lib/github_secret_scanning.rb

```diff
@@ -0,0 +1,29 @@
+class GitHubSecretScanning
```

Comment:
> E, [2021-04-01T03:55:04.081090 #62] ERROR -- : uninitialized constant GithubSecretScanning                                                            
Did you mean?  GitHubSecretScanning (NameError) 

You should either add a underscore in file name or use class name with small h (in GitHub)

Reply:
I thought I had added the necessary 'require' statement as a initializer, that would replace the need for the autoload. How did you get the problem?

Reply:
I was checking out the build failure for the docker step. You can reproduce by running `RAILS_ENV=production SECRET_KEY_BASE=1234  unicorn_rails -E production` or docker run command from `build_docker.sh`. Seem to be coming from eager loading.
```
E, [2021-04-02T20:06:47.588498 #59618] ERROR -- : uninitialized constant GithubSecretScanning                                                         
Did you mean?  GitHubSecretScanning (NameError)                                                                                                       
/home/aditya/.rvm/gems/ruby-2.7.2/gems/zeitwerk-2.4.2/lib/zeitwerk/loader.rb:409:in `const_get'                                                       
/home/aditya/.rvm/gems/ruby-2.7.2/gems/zeitwerk-2.4.2/lib/zeitwerk/loader.rb:409:in `block (2 levels) in eager_load'                                  
/home/aditya/.rvm/gems/ruby-2.7.2/gems/zeitwerk-2.4.2/lib/zeitwerk/loader.rb:733:in `block in ls'                                                     
/home/aditya/.rvm/gems/ruby-2.7.2/gems/zeitwerk-2.4.2/lib/zeitwerk/loader.rb:725:in `foreach'                                                         
/home/aditya/.rvm/gems/ruby-2.7.2/gems/zeitwerk-2.4.2/lib/zeitwerk/loader.rb:725:in `ls'                                                              
/home/aditya/.rvm/gems/ruby-2.7.2/gems/zeitwerk-2.4.2/lib/zeitwerk/loader.rb:404:in `block in eager_load'                                             
/home/aditya/.rvm/gems/ruby-2.7.2/gems/zeitwerk-2.4.2/lib/zeitwerk/loader.rb:393:in `synchronize'                                                     
/home/aditya/.rvm/gems/ruby-2.7.2/gems/zeitwerk-2.4.2/lib/zeitwerk/loader.rb:393:in `eager_load'                                                      
/home/aditya/.rvm/gems/ruby-2.7.2/gems/zeitwerk-2.4.2/lib/zeitwerk/loader.rb:508:in `each'                                                            
/home/aditya/.rvm/gems/ruby-2.7.2/gems/zeitwerk-2.4.2/lib/zeitwerk/loader.rb:508:in `eager_load_all' 
```